purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1242,6 +1239,13 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         @JvmStatic
         var platformFlavor = "native"
 
+        /**
+         * DO NOT MODIFY. This is used internally by the Hybrid SDKs to indicate which platform version
+         * is being used
+         */
+        @JvmStatic
+        var platformFlavorSDKVersion: String? = null
```

Comment:
since we should never have one of these two without the other, and we don't need the values to be readable from the outside, would it make sense to encapsulate setting them in a single set method?

Reply:
Good point. I actually mentioned that before but never got to do it. I am going to make the changes